config/services/wivrn.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>wivrn</short>
+  <description>
+    The Linux OpenXR streaming application to standalone headsets 
+  </description>
```

Comment:
Looking at other service files, e.g. https://github.com/firewalld/firewalld/blob/05586b0550894f886126905bbb8bba8f4e6536c4/config/services/alvr.xml, I believe the text could be a bit longer:
```suggestion
  <short>WiVRn wirelessly connects a standalone VR headset to a Linux computer</short>
  <description>
    Play PCVR games on the headset while processing is done on the computer. It supports a wide range of headsets such as Quest 1 / 2 / Pro / 3 / 3S, Pico Neo 3, Pico 4, HTC Vive Focus 3, HTC Vive XR elite and most other Android based headsets.
  </description>
```